crates/ui/src/dock/tiles_panel.rs

```diff
@@ -0,0 +1,728 @@
+                                y: item.bounds.origin.y + item.bounds.size.height
+                                    - px(HALF_HANDLE_SIZE),
+                            },
+                            Size {
+                                width: px(HANDLE_SIZE),
+                                height: px(HANDLE_SIZE),
+                            },
+                        );
+
+                        let drag_bar_bounds = Bounds::new(
+                            item.bounds.origin,
+                            Size {
+                                width: item.bounds.size.width,
+                                height: px(DRAG_BAR_HEIGHT),
+                            },
+                        );
+
+                        v_flex()
+                            .border_1()
+                            .rounded_md()
+                            .border_color(border)
+                            .absolute()
+                            .left(item.bounds.origin.x)
+                            .top(item.bounds.origin.y)
+                            .w(item.bounds.size.width)
+                            .h(item.bounds.size.height)
+                            .child(
+                                h_flex()
+                                    .w_full()
+                                    .h_full()
+                                    .overflow_hidden()
+                                    .child(panel_view),
+                            )
+                            .child(if !is_occluded(&right_handle_bounds) {
+                                div()
+                                    .id("right-resize-handle")
+                                    .cursor_col_resize()
+                                    .absolute()
+                                    .top(px(0.0))
+                                    // Offset is -HALF_HANDLE_SIZE
+                                    .right(px(-HALF_HANDLE_SIZE))
+                                    .w(px(HANDLE_SIZE))
+                                    .h(item.bounds.size.height)
+                                    .on_mouse_down(
+                                        MouseButton::Left,
+                                        cx.listener(move |this, event: &MouseDownEvent, cx| {
+                                            let initial_mouse_position = event.position;
+                                            let panel_bounds = item.bounds;
+                                            let drag_data = ResizeDragData {
+                                                axis: ResizeAxis::Horizontal,
+                                                initial_mouse_position,
+                                                initial_panel_bounds: panel_bounds,
+                                            };
+                                            this.update_resizing_drag(drag_data, cx);
+                                            this.bring_panel_to_front(this.resizing_panel_index);
```

Comment:
`resizing_panel_index` -> `resizing_index`

In the `tiles`, the main item is the `panel` not have any other things.

So `active_index`, `resizing_index` that must means the `panel`.